cmd/up/up.go

```diff
@@ -975,9 +931,7 @@ func printDisplayContext(dev *model.Dev) {
 	}
 	log.Println(fmt.Sprintf("    %s %s", log.BlueString("Namespace:"), dev.Namespace))
 	log.Println(fmt.Sprintf("    %s      %s", log.BlueString("Name:"), dev.Name))
-	if dev.RemoteModeEnabled() {
-		log.Println(fmt.Sprintf("    %s       %d -> %d", log.BlueString("SSH:"), dev.RemotePort, dev.SSHServerPort))
-	}
+	log.Println(fmt.Sprintf("    %s       %d -> %d", log.BlueString("SSH:"), dev.RemotePort, dev.SSHServerPort))
```

Comment:
@rberrelleza would you remove this line? at the end, it is more like an implementation detail

Reply:
I guess. but we should think about documenting how to access the dev env with SSH tools too (there's an issue opened on showing the path to the SSH key we use for this)